js/scenes/MyProjects/MyProjects.js

```diff
@@ -1,11 +1,39 @@
 import React from "react";
-import { Text, View } from "react-native";
+import { Image, Text, View } from "react-native";
 import PropTypes from "prop-types";
+import Icon from "react-native-vector-icons/Ionicons";
 import { styles } from "./styles";
+import { colors, colorShadow } from "../../config/styles";
+
+// import Nova UI components:
+import NovaH2 from "../../components/UI/NovaH2";
+import NovaButton from "../../components/UI/NovaButton";
+
+const { red } = colors;
 
 const MyProjects = () => (
   <View>
-    <Text>MyProjects Scene</Text>
+    <View style={styles.headerContainer}>
+      <View style={styles.header}>
+        <View style={colorShadow}>
+          <Image
+            style={styles.logo}
+            source={{
+              uri:
+                "https://firebasestorage.googleapis.com/v0/b/nova-2018.appspot.com/o/company-logos%2Fcompany-logo-deloitte.png?alt=media&token=e1fa5bfb-9316-400a-bc43-d02267907f64"
```

Comment:
should not use hardcoded path. download that image and use require instead of uri.